beep/helpers/featurizer_helpers.py

```diff
@@ -406,51 +505,154 @@ def get_V_I(df):
     result['current'] = current
     return result
 
-
-def get_v_diff(diag_num, processed_cycler_run, soc_window):
+def get_v_diff(processed_cycler_run, diag_pos, soc_window):
     """
-    This function helps us get the feature of the variance of the voltage difference
-    across a specific capacity window
-    Argument:
-            diag_num(int): diagnostic cycle number at which you want to get the feature, such as 37 or 142
-            processed_cycler_run(process_cycler_run object)
-            soc_window(int): let the function know which step_counter_index you want to look at
+    This function calculates voltage difference between a hppc cycle we are interested (at diag_pos) and the initial
+    diag cycle (cycle 2) for a specific soc range
+
+    Args:
+        processed_cycler_run (beep.structure.ProcessedCyclerRun)
+        diag_pos (int): diagnostic cycle occurence for a specific <diagnostic_cycle_type>. e.g.
+        if rpt_0.2C, occurs at cycle_index = [2, 42, 147, 249 ...], <diag_pos>=0 would correspond to cycle_index 2
+        soc_window (int): step index counter corresponding to the soc window of interest.
+
     Returns:
-            a float
+        a dataframe that contains this a value (float), the variance of the voltage differences
```

Comment:
Should the return be a dataframe containing the variance of voltage differences?